### DISCUSSION TITLE
Duda entrega MacoWins

### DISCUSSION THREAD

**Discussion by Schmidt-mat** [dds-jv/foro#343] [Consultas generales] [answered]
Buenas, tengo una duda con respecto al tag de la entrega de MacoWins. Nos piden que creemos el tag "entrega-final" pero en mi caso ese tag ya está creado en el repo, ¿lo borro y lo vuelvo a crear o creo otro con un nombre similar a entrega-final?

**Comment by gastonprieto** [comment, marked as answer]
Buenas @Schmidt-mat 

Borralo y crealo de nuevo, en este repo hay un par de discusiones que explican como.

Igualmente para próximas entregas fijate bien antes de generar el tag

**Reply by gastonprieto** [reply]
esto te sirvio @Schmidt-mat ?

**Reply by Schmidt-mat** [reply]
Sisi, borré el tag y lo volví a crear, gracias.

**Reply by Schmidt-mat** [reply]
Lo que pasó es que por algún motivo cuando hago los comandos mvn clean verify && git tag entrega-final && git push origin HEAD --tags el tag no se asocia al último commit, sino que me toma un commit anterior, por eso pensaba que ya venía creado en el repo, me aparecía que había sido de hace varios días el tag pero en realidad lo que pasó es que el comando de git push origin HEAD --tags no me anda bien, se estaba asociando el tag al primer commit, el de la creación del repo. Para solucionarlo hice push y después cree el tag.
